### ISSUE TITLE
Tags aren't focusable at all by keyboard

### ISSUE THREAD

**Issue by lukebarnard1** [vector-im/element-web#6186]

**Comment by lampholder** [comment]
Is this easy to fix?

**Comment by lampholder** [comment]
By which I mean, unless it's really _difficult_ to fix it would nice not to ship LLP with obvious accessibility annoyances (though I realise I don't know whether we've considered any other aspects of accessibility here yet).

**Comment by lukebarnard1** [comment]
`react-beautiful-dnd` claims to be super accessible (you should be able to move things with the keyboard) so we're clearly doing something wrong here.

**Comment by ara4n** [comment]
Have renamed bug to reflect current behaviour; this is by design; it will all go away with the new redesign.
